mne_bids/tests/test_write.py

```diff
@@ -2851,6 +2851,50 @@ def test_convert_eeg_formats(dir_name, format, fname, reader, tmp_path):
         raw.get_data(), raw2.get_data()[:, :orig_len], decimal=6)
 
 
+@requires_version('mne', '0.24')
+@requires_version('pybv', '0.6')
+@pytest.mark.parametrize(
+    'dir_name, format, fname, reader', test_converteeg_data)
+@pytest.mark.filterwarnings(
+    warning_str['channel_unit_changed'], warning_str['edfblocks'])
+def test_convert_multiple(dir_name, format, fname, reader, tmp_path):
+    bids_root = tmp_path / format
```

Comment:
Can you add a tiny docstring here, explaining what is being tested?

Reply:
Done.